### DISCUSSION TITLE
Receiving Voicemail link through Omnichannel in Salesforce

### DISCUSSION THREAD

**Discussion by shaaju** [amazon-connect/amazon-connect-salesforce-scv#100] [Q&A] [answered]
I have implemented the VMX 2 through AWS Tasks. The link to download the voicemail is successfully received on the Agent CCP on Amazon Connect. However, in Salesforce, I just hear ringing but there is nothing shown on Omni-channel tool. Do I need to enable some options to receive tasks created for Voicemail on Omni-channel tool?

**Comment by dougjaso** [comment, marked as answer]
AWS Tasks is not supported as a delivery mode in Service Cloud Voice. Omnichannel does not recognize Connect tasks and will not present them to the agent. For SCV, only case and custom object are supported for SCV bundle accounts. For SCV - Partner Telephony with Amazon Connect, you can also use email.

**Reply by shaaju** [reply]
Thanks for clarifying this. I have a follow-up question: For voicemail as email, the requirement is that agent's ID must be their email address. However, when adding Agents in Service Cloud Voice, the Agents created in Amazon Connect get somewhat random login ID. Is this the ID being referred to? If yes, can it be changed to email address without any consequences to the mapping between SCV and Amazon Connect users?

**Reply by dougjaso** [reply]
Service cloud voice Bundle will not support Voicemail as email. Amazon Simple Email Service is not available in SCV bundle accounts. Additionally, you cannot change the SCV login IDs.
